docs/releases/2.0.txt

```diff
@@ -163,6 +163,9 @@ Minor features
 * :djadmin:`inspectdb` can now introspect ``JSONField`` and various
   ``RangeField``\s (``django.contrib.postgres`` must be in ``INSTALLED_APPS``).
 
+* The ``in`` lookup for :class:`~django.contrib.postgres.fields.array.ArrayField`
+  supports subqueries
```

Comment:
Last thing I can see, you still need to add a period (.) at the end of the sentence 😉